frontend/src/components/adminPage/ProfessionRows.js

```diff
@@ -89,63 +113,84 @@ import '../../style/ProfessionRows.css';
+            </div>
+            </div>
+            <div className="col-12 col-md-10 mt-1" id="RateCoeffTable">
+            <div className="row professionrow">
+            <div className="col-3 text-center" id="col-custom"></div>
+            <div className="col-4 text-center" id="col-custom">
                 
-                    {idDoc === null ? "Rate" : "Coefficient"}
+                    {idDoc === null ? "Rate" : "Coeff"}
                 
             </div>
-            <div className="col-6 text-center" id="col-custom dateDiv">
+            <div className="col-5 text-center" id="col-custom dateDiv">
                 Date
                 </div>
             </div>
             {
             this.state.ratesArr.map(
             rate =>
-            <div className="row professionrow">
-                <div className="col-6" id="col-custom">
-                    <div className="row col-xs-12 col-sm-12 col-md-12">
-                        <div className="col-8">
-                            {
-                                idDoc === null ? rate.Rate : rate.Coeff
-                            }
-                        </div> 
+            <div className="row professionrow" id={"salaryinfo" + rate.StartDate}>
+                <div className="col-3" id="col-custom">
                         {
                         rate.State==1?
                         images =
-                        <div className="col-3 mt-2" >
-                            <i className="fa fa-pencil-alt col-1 mr-2 ml-4" data-toggle="modal" data-target="#EditRateToProfession" onClick = {() => this.changeCurrentDate(rate.StartDate)}></i>
-                            <i className="fa fa-times col-1" data-toggle="modal" data-target="#DeleteRateToProfession" onClick = {() => this.changeCurrentDate(rate.StartDate)}></i>  
+                        <div className=" row mt-2" >
+                            <i id="edit" className="fa fa-pencil-alt ml-2" data-toggle="modal" data-target="#EditRateToProfession" onClick = {() => this.changeCurrentDate(rate.StartDate)}></i>
+                            <i id="delete" className="fa fa-times ml-2" data-toggle="modal" data-target="#DeleteRateToProfession" onClick = {() => this.changeCurrentDate(rate.StartDate)}></i>  
                         </div>
                         : (rate.State==0?
-                            images =
-                            <div className="col-3 mt-2" >
-                                <i className="fa fa-pencil-alt col-1 ml-4" data-toggle="modal" data-target="#EditRateToProfession" onClick = {() => this.changeCurrentDate(rate.StartDate)}></i>
+                            (images =
+                            <div className=" row mt-2" >
+                                <i id="edit" className="fa fa-pencil-alt ml-2" data-toggle="modal" data-target="#EditRateToProfession" onClick = {() => this.changeCurrentDate(rate.StartDate)}></i>
```

Comment:
Each page can have only one element with that "id", for multiple elements use "class"